base/Dockerfile

```diff
@@ -0,0 +1,213 @@
+
+# Setup work directory for backward-compatibility
+RUN mkdir "/home/${NB_USER}/work" && \
+    fix-permissions "/home/${NB_USER}"
+
+# Download and install Micromamba, and initialize Conda prefix.
+#   <https://github.com/mamba-org/mamba#micromamba>
+#   Similar projects using Micromamba:
+#     - Micromamba-Docker: <https://github.com/mamba-org/micromamba-docker>
+#     - repo2docker: <https://github.com/jupyterhub/repo2docker>
+# Install Python, Mamba and jupyter_core
+# Cleanup temporary files and remove Micromamba
+# Correct permissions
+# Do all this in a single RUN command to avoid duplicating all of the
+# files across image layers when the permissions change
+COPY --chown="${NB_UID}:${NB_GID}" initial-condarc "${CONDA_DIR}/.condarc"
+WORKDIR /tmp
+RUN set -x && \
+    arch=$(uname -m) && \
+    if [ "${arch}" = "x86_64" ]; then \
+        # Should be simpler, see <https://github.com/mamba-org/mamba/issues/1437>
+        arch="64"; \
+    fi && \
+    wget --progress=dot:giga -O /tmp/micromamba.tar.bz2 \
+        "https://micromamba.snakepit.net/api/micromamba/linux-${arch}/latest" && \
+    tar -xvjf /tmp/micromamba.tar.bz2 --strip-components=1 bin/micromamba && \
+    rm /tmp/micromamba.tar.bz2 && \
+    PYTHON_SPECIFIER="python=${PYTHON_VERSION}" && \
+    if [[ "${PYTHON_VERSION}" == "default" ]]; then PYTHON_SPECIFIER="python"; fi && \
+    # Install the packages
+    ./micromamba install \
+        --root-prefix="${CONDA_DIR}" \
+        --prefix="${CONDA_DIR}" \
+        --yes \
+        "${PYTHON_SPECIFIER}" \
+        'mamba' \
+        'jupyter_core' && \
+    rm micromamba && \
+    # Pin major.minor version of python
+    mamba list python | grep '^python ' | tr -s ' ' | cut -d ' ' -f 1,2 >> "${CONDA_DIR}/conda-meta/pinned" && \
+    mamba clean --all -f -y && \
+    fix-permissions "${CONDA_DIR}" && \
+    fix-permissions "/home/${NB_USER}"
+
+# Configure container startup
+ENTRYPOINT ["tini", "-g", "--"]
+
+# Copy local files as late as possible to avoid cache busting
+COPY run-hooks.sh start.sh /usr/local/bin/
+
+USER root
+
+# Create dirs for startup hooks
+RUN mkdir /usr/local/bin/start-notebook.d && \
+    mkdir /usr/local/bin/before-notebook.d
+
+# ------------------------------------
+#            BASE NOTEBOOK
+# ------------------------------------
+
+# This part of the base image is based on the upstream base-notebook
+# image and files, available GitHub: 
+# https://github.com/jupyter/docker-stacks/tree/0d8b4e498766fb451e3ba08f9ade9da344db5d83/images/base-notebook
+
+# Install all base notebook dependencies
+RUN dnf install -y pandoc
+
+# Switch back to jovyan to avoid accidental container runs as root
+USER ${NB_UID}
+
+# Install JupyterLab, Jupyter Notebook, JupyterHub and NBClassic
+# Generate a Jupyter Server config
+# Cleanup temporary files
+# Correct permissions
+# Do all this in a single RUN command to avoid duplicating all of the
+# files across image layers when the permissions change
+WORKDIR /tmp
```

Comment:
Missing the comments on why we cannot use latest notebook and why, in that case, we still install the old, given that swanclassic doesn't need it.

Reply:
I added a comment regarding the incompatibility issue and I also added it to the commit message!

Reply:
I think you committed this with the wrong commit.

Reply:
Also, the reason we still need notebook is because of the nb extensions configurator, right? However, we saw some log errors so we might even need to remove it.

Reply:
What are you referring to as being committed in the wrong commit?

Reply:
Yes, exactly. If we remove the nbextensions configurator it should not be a problem anymore

Reply:
> What are you referring to as being committed in the wrong commit?

It looked wrong, but now I see all is fine. (although, technically, you changed the version the less than 7 in a previous commit :) )

Reply:
> > What are you referring to as being committed in the wrong commit?
> 
> It looked wrong, but now I see all is fine. (although, technically, you changed the version the less than 7 in a previous commit :) )

Sorry, no.. it's wrong... 
https://github.com/swan-cern/jupyter-images/pull/2/commits/6661df9c556fad40b562cbc3edfa40061ee8daf7

This commit shows the adding of the comment about the notebook version. But the title of this commit is "Add SWAN section to include swanheader package"